forbidden.html

```diff
@@ -0,0 +1,101 @@
+<!doctype html>
+<html class="no-js" lang="en-US" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+
+    <title> sso.mozilla.com | forbidden</title>
+
+    <link rel="stylesheet" href="/media/css/sandstone/sandstone.css" media="all">
+
+    <link href="//mozorg.cdn.mozilla.net/media/css/tabzilla-min.css" rel="stylesheet">
+
+    <link rel="shortcut icon" href="/media/img/favicon.ico">
+
+    <!--[if lte IE 8]>
+    <script src="/media/js/libs/html5shiv.js"></script>
+    <![endif]-->
+
+  </head>
+
+  <body class="sand">
+
+    <div id="outer-wrapper">
+      <div id="wrapper">
+
+        <header id="masthead">
+          <a href="https://www.mozilla.org/" id="tabzilla" data-infobar="update">Mozilla</a>
+
+          <!--
+            The main navigation is optional, for pages that need it.
+            The number of links should be kept to a minimum, no more than 5 or 6.
+            If you need more than 6 links in this navbar, you have bigger IA problems to focus on.
+          -->
+          <nav id="nav-main" role="navigation">
+            <span class="toggle" role="button" aria-controls="nav-main-menu" tabindex="0">Menu</span>
+          </nav>
+
+          <h2><a href="http://www.mozilla.org"><img src="/media/img/sandstone/header-mozilla-stone.png" alt="mozilla"></a></h2>
+
+          <!--
+            Breadcrumbs are optional, if the hierarchy warrants it.
+            The number of links should be kept to a minimum, no more than 3 or 4.
+            If you need more than 4 breadcrumb links, you have bigger IA problems to focus on.
+          -->
+        </header>
+
+        <!--
+          The main feature of a page is usually the main title, but may also consists of a promotional
+          banners, introductory text, etc. However, this should not be the main content or "meat" of the
+          page, so it shouldn't contain a lot of text or a full article. Think of it as a masthead for
+          this specific page below the global masthead for the entire site. Use an appropriate element
+          with the id "main-feature". This might be a single heading (h1), an hgroup, a div, etc.
+        -->
+        <hgroup id="main-feature" class="center">
+          <br /><br /><br /><br />
+          <h2>You are not authorized</h2><br /><br />
+          Though you logged in successfully, unfortunately, you aren't authorized to access this service.
```

Comment:
Consider adding a sentence guiding them on how to request help regarding access issues, so that we can funnel those issues into the correct bin.